data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg

```diff
@@ -305,26 +310,75 @@
+            message= _ "At the start of each turn, allied units adjacent to a Shaman will recover 4 hitpoints. This is only half of the amount that a village would provide, but you rarely have a village near your front-line. The Shaman doesn’t gain experience from healing, only from combat, but advancing her is worthwhile — her second-level Elvish Druid advancement has healing equivalent to a village.
+
+A detailed explanation of healing may be found in the help (default hotkey: <b>F1</b>), in the <i>Gameplay</i> section."
         [/message]
 
         [message]
             speaker=narrator
-            caption= _ "Abilities and Weapon Specials"
+            caption= _ "Weapon Specials"
             image=wesnoth-icon.png
-            message= _ "Weapon specials are special attack modifiers that make certain attacks more powerful. Once again, in this case, the Shaman has the Slows special on her entangle attack. When this attack is used on an enemy, it will halve the damage they do for one turn. Other weapon specials include First Strike, which allows that attack to always hit first no matter who attacks first, and Magical, which gives that attack a constant 70% chance to hit.
+            message= _ "Weapon specials are special attack modifiers that make certain attacks more powerful. In this case, the Shaman has the Slows special on her entangle attack. When this attack successfully hits an enemy, it will halve the damage they do for one turn. Other weapon specials include First Strike, which allows that attack to always hit first no matter who attacks first, and Magical, which gives that attack a constant 70% chance to hit.
 
 A full list of abilities and weapons specials, along with traits, may be found in the help (default hotkey: <b>F1</b>)."
         [/message]
     [/event]
 
+    # Triggers the first time that the player has two healers.
+    # If there are allied sides then this isn't accurate, but at this stage of the tutorial allied sides haven't been introduced yet.
+    [event]
+        name=recruit
+        [filter]
+            side=1
+            type=Elvish Shaman
+        [/filter]
+        [filter_condition]
+            [have_unit]
+                ability_type=heals
+                count=2
+            [/have_unit]
+        [/filter_condition]
+
+        [if]
+            [have_unit]
+                type=Elvish Shaman
+                count=2
+            [/have_unit]
+            [then]
+                [message]
+                    speaker=unit
+                    # po: the speaker and the other healer are both level one elvish shamans
+                    message= _ "You now have two healers. However, each unit can only receive healing from one healer — even if we’re both next to a unit then that unit will still only be healed 4 hitpoints per turn.
```

Comment:
```suggestion
                    message= _ "Shaman number two, reporting in! I can heal units just like my friend, but only one of us can work at a time — even if we’re both next to a unit we can only heal them 4 hitpoints per turn.
```

Reply:
The "reporting in" sounds wrong to me.

The "only one of us can work at a time" seems more confusing to me, particularly when there are other troops next to the shamans. Consider 5 units in a line, injured, shaman, injured, shaman, injured.

Reply:
Could we make it the narrator saying this line then?

Reply:
We could do, but why?

Reply:
The way it's written, it doesn't sound very much like character dialogue to me. There's very little style to it, so I think that's better off for an instructor (Galdrad) or better yet, a narrator.